File: python/sumuj2.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  sumuj2.py


def main(args):
    suma = 0
    ile = 0
    while True:
        liczba = int(input('Podaj liczbe: '))
        suma = suma + liczba
        ile = ile+ 1
        if suma > 100:
            break
    print('wprowadzona', ile, 'liczb.')
    print('Suma: ', suma)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/sumuj2.cpp
/*
 * sumuj2.cpp
 * Program pobiera liczby od urzytkownika aż ich suma przekroczy 100.
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	// int i;
    int suma = 0;
    int liczba = 0;
    int ilosc_liczb = 0;
    cout << "Podaj ilość liczb: " << endl;
    cout << "Wprowadzaj kolejne liczby: " <<endl;
    // for (;;)
    while (1)
    {
        cin >> liczba;
        ilosc_liczb++;
        cout << "Podano: " << liczba << endl;
        suma += liczba;
        
        if (suma > 100)
            break;
    
    };
    cout << "Suma liczb: " << suma << endl;
    cout << "Wprowadż liczb: " << ilosc_liczb << endl; 
    
        
	return 0;
}

